test/quantization/core/test_quantized_op.py

```diff
@@ -3814,6 +3814,111 @@ def test_qlinear_unpack(self, W, use_channelwise):
             np.testing.assert_equal(
                 W_q.q_zero_point(), W_q_origin.q_zero_point())
 
+    @given(batch_size=st.integers(1, 4),
+           input_channels=st.integers(16, 32),
+           output_channels=st.integers(4, 8),
+           use_bias=st.booleans(),
+           use_multi_dim_input=st.booleans(),
+           use_channelwise=st.booleans())
+    @skipIfNoONEDNN
+    def test_qlinear_leaky_relu(self, batch_size, input_channels, output_channels, use_bias,
+                                use_multi_dim_input, use_channelwise):
+        # Test numerical correctness of qlinear_leaky_relu
+        # Pass 1: X_q, W_q -> qlinear_leaky_relu -> Y_q
+        # Pass 2: X_q, W_q -> dequantize -> linear -> leaky_relu -> quantize -> Y_q_ref
+        # Compare the results of the two passes
+        decimal_val = 4
+        dtype = torch.quint8
+        negative_slope = 0.01
+        # for onednn backend only
+        with override_quantized_engine('onednn'):
```

Comment:
are we still planning to use onednn, or do we want to switch to use x86 now?

Reply:
Thanks for point, @jerryzh168 . Ideally, we'd enable these new fusions on x86 directly. But the problem now is that the perf with onednn kernel cannot always be better than fbgemm even with post-op fusion. Therefore, our plan is to enable these new fusions on onednn backend first (which can benefit users in some cases but not all) while perf of onednn library being improved. Then, these new fusions can be moved to x86 when onednn kernel plus fusion can always bring better perf. Does this plan make sense to you?